app/main.py

```diff
@@ -1 +1,41 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        if self not in Animal.alive:
```

Comment:
Always append new animals to the `alive` class attribute, it is unlikely that `__init__` would be called on an instance that was initialized earlier